app/assets/stylesheets/youtube.sass

```diff
@@ -0,0 +1,6 @@
+.youtube-player
+  position: absolute
+  top: 0
+  right: 0
+  bottom: 0
+  left: 0
```

Comment:
Since this is the exact same as `.twitch-player`, maybe we can merge them into one `.video-player` class, or at least into one file together if you think that's too aggressive.

Reply:
That sounds fine. I considered it originally but didn't as that would affect the existing Twitch JS code, but it shouldn't be a huge deal to tweak it to be able to figure out if we need to load the Twich player vs the YouTube player.

Reply:
Ohh then nevermind, I didn't realize `.twitch-player` was being used to select elements, thought it was just for this CSS here. No worries then.

Reply:
If it's just being used for element selection, we could always just use something like `data-youtube` and `data-twitch` attributes.

Reply:
The CSS has been combined and it now uses `data` attributes to do the correct thing for element selection.